app/types.ts

```diff
@@ -123,4 +123,18 @@ export interface Teacher {
     first_name: string,
     last_name: string
   }
-}
\ No newline at end of file
+}
+
+export interface Quiz {
```

Comment:
type should be QuizSession I feel, instead of Quiz. 

Quiz --> is decoupled with start/end times. It exists on mongoDb. Firestore content would correspond to a quiz session

Reply:
Updated!